<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="385" total="386" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-28T18:34:44 IST" name="Suite" finished-at="2025-02-28T18:36:41 IST" duration-ms="116712">
    <groups>
    </groups>
    <test started-at="2025-02-28T18:34:44 IST" name="Test" finished-at="2025-02-28T18:36:41 IST" duration-ms="116712">
      <class name="commonMethods.Testcases">
        <test-method is-config="true" signature="setUp()[pri:0, instance:commonMethods.Testcases@388d14e]" started-at="2025-02-28T18:34:44 IST" name="setUp" finished-at="2025-02-28T18:34:44 IST" duration-ms="408" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="Extentreport()[pri:0, instance:commonMethods.Testcases@388d14e]" started-at="2025-02-28T18:34:44 IST" name="Extentreport" finished-at="2025-02-28T18:34:44 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Extentreport -->
        <test-method is-config="true" signature="getDataFromConfig()[pri:0, instance:commonMethods.Testcases@388d14e]" started-at="2025-02-28T18:34:44 IST" name="getDataFromConfig" finished-at="2025-02-28T18:34:44 IST" duration-ms="35" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDataFromConfig -->
        <test-method signature="TC_Rate_Request_SC01()[pri:0, instance:commonMethods.Testcases@388d14e]" started-at="2025-02-28T18:34:44 IST" name="TC_Rate_Request_SC01" finished-at="2025-02-28T18:36:41 IST" duration-ms="116621" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test failed due to error: ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test failed due to error: 
at org.testng.Assert.fail(Assert.java:99)
at commonMethods.Testcases.runTestForDataset(Testcases.java:722)
at commonMethods.Testcases.TC_Rate_Request_SC01(Testcases.java:1598)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_Rate_Request_SC01 -->
        <test-method is-config="true" signature="Teardown(org.testng.ITestResult)[pri:0, instance:commonMethods.Testcases@388d14e]" started-at="2025-02-28T18:36:41 IST" name="Teardown" finished-at="2025-02-28T18:36:41 IST" duration-ms="8" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TC_Rate_Request_SC01 status=FAILURE method=Testcases.TC_Rate_Request_SC01()[pri:0, instance:commonMethods.Testcases@388d14e] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method is-config="true" signature="aftertest()[pri:0, instance:commonMethods.Testcases@388d14e]" started-at="2025-02-28T18:36:41 IST" name="aftertest" finished-at="2025-02-28T18:36:41 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- commonMethods.Testcases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
